bin/ruby-build

```diff
@@ -1,14 +1,16 @@
 #!/usr/bin/env bash
 #
-# Usage: ruby-build [-kvp] <definition> <prefix>
+# Usage: ruby-build [-kpv46] <definition> <prefix>
```

Comment:
I believe they "work", but I'm just not sure what they are doing and when should they be used.